perfkitbenchmarker/benchmarks/cloudsuite_web_serving.py

```diff
@@ -0,0 +1,368 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs Cloudsuite2.0 Web Serving benchmark
+   More info: http://parsa.epfl.ch/cloudsuite/web.html
+"""
+import posixpath
+import re
+
+from perfkitbenchmarker import sample
+from perfkitbenchmarker import vm_util
+from perfkitbenchmarker import flags
+from perfkitbenchmarker.packages import nginx
+from perfkitbenchmarker.packages import php
+
+# environment variables
+BASE_DIR = posixpath.join(vm_util.VM_TMP_DIR, 'web-release')
+CATALINA_HOME = '%s/apache-tomcat-6.0.35' % BASE_DIR
+OLIO_HOME = '%s/apache-olio-php-src-0.2' % BASE_DIR
+FABAN_HOME = '%s/faban' % BASE_DIR
+MYSQL_HOME = '%s/mysql-5.5.20-linux2.6-x86_64' % BASE_DIR
+GEOCODER_HOME = '%s/geo' % BASE_DIR
+APP_DIR = '%s/app' % BASE_DIR
+JAVA_HOME = '$(readlink -f $(which java) | cut -d "/" -f 1-5)'
+PHPRC = posixpath.join(APP_DIR, 'etc')
+OUTPUT_DIR = posixpath.join(vm_util.VM_TMP_DIR, 'out1')
+ANT_HOME = posixpath.join(vm_util.VM_TMP_DIR, 'ant/bin')
+OLIO_BUILD = '%s/apache-olio-php-src-0.2/workload/php/trunk/' % BASE_DIR
+# File names
+MY_CNF = '/etc/my.cnf'
+NGINX_CONF = '%s/nginx.conf' % BASE_DIR
+CL = 0
+BK = 1
+FR = 2
+
+flags.DEFINE_integer('load_scale', 100,
+                     'The maximum number of concurrent users '
+                     'that can be simulated.', lower_bound=2)
+FLAGS = flags.FLAGS
+
+BENCHMARK_INFO = {'name': 'cloudsuite_web_serving',
```

Comment:
RE: name of the benchmark module, other files in this directory end in '_benchmark.py'